io/disk/ssd/blkdiscard.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2016 IBM
+# Author: Venkat Rao B <vrbagal1@linux.vnet.ibm.com>
+
+# Blkdiscard  is  used  to  discard  device  sectors.This is useful for
+# solid-state drivers (SSDs) and thinly-provisioned storage.
+
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import process
+from avocado import main
+
+
+class Blkdiscard(Test):
+
+    """
+    blkdiscard is used to discard device sectors.This is useful for
+    solid-state drivers (SSDs) and thinly-provisioned storage.
+    """
+
+    def setUp(self):
+
+        """
+        Checks if the blkdiscard packages are installed or not.
+        """
+        smm = SoftwareManager()
+        if not smm.check_installed("util-linux"):
+            self.skip("blkdiscard is needed for the test to be run")
+        self.disk = self.params.get('self.disk', default='/dev/nvme0n1')
```

Comment:
Seems like it should be `self.params.get('disk', default='/dev/nvme0n1')`, right?